crates/holochain_state/src/validation_db.rs

```diff
@@ -121,7 +121,7 @@ mod tests {
             let op = op.clone();
             let op_hashed = DhtOpHashed::from_content_sync(op);
             let test_stage = test_stage.clone();
-            let test_validation_status = test_validation_status.clone();
+            let test_validation_status = test_validation_status;
```

Comment:
I don't think this is necessary at this point

Reply:
Yep, you're right